internal/util/paths/paths.go

```diff
@@ -79,17 +78,24 @@ func WriteString(path string, content string) error {
 	return err
 }
 
-// Expands leading tilde.
-func ExpandTilde(path string) (string, error) {
+// Expands home directory to absolute path.
+// If no expansion is required or possible, the string is returned unaltered.
+// If there is an error, an empty string is returned.
+func ExpandHomeDir(path string) (string, error) {
```

Comment:
should this just be called `Expand`? You're expanding the `~` and any environment variables

Reply:
I would suggest `ExpandTileAndEnv`, imho `Expand` is not specific enough.

Reply:
Or `ExpandPath`? The `..Env` part could also mean an "environment" in some other context than a shell environment variable? It's reasonable to understand that `ExpandPath` would include tilde expansion?

```go
// Expands environment variables and `~`, returning an absolute path.
func ExpandPath(path string) (string, error) {}
```